app/subscriber/src/features/my-reports/edit/settings/template/utils/getChartData.ts

```diff
@@ -50,9 +50,21 @@ export const getChartData = (
   // Extract unique values to map to the axis labels.
   const groups =
     options.groups ??
-    getDistinct(content, (d) => options.groupOn(d)).map((s) => options.groupOn(s));
+    getDistinct(content, (d) => options.groupOn(d))
+      .map((s) => options.groupOn(s))
+      .sort((a, b) => {
```

Comment:
Sort UI alphabetically.  This will most likely need to change and follow the configured sort for the datasource at some point.